ocean_provider/routes/compute.py

```diff
@@ -220,6 +220,7 @@ def computeStatus():
 
         _response = response.content
 
+        # The following sections is needed only to ensure back compat. It will be removed soon, make sure you are going to update your C2D backends
```

Comment:
can you add this to the Release Notes when we upgrade the version? As a future deprecation?